webapp/lib/app/configurator/standard_messages/controller.dart

```diff
@@ -202,8 +238,7 @@ class MessagesConfiguratorController extends ConfiguratorController {
       await Future.wait([platform.updateSuggestedReplies(diffData.editedMessages), platform.deleteSuggestedReplies(diffData.deletedMessages)]);
       _view
         ..unsavedChanges = false
-        ..showSaveStatus('Saved!', autoHide: true)
-        ..clearUnsavedIndicators();
+        ..showSaveStatus('Published for saving!', autoHide: true);
```

Comment:
Nit: "published for saving" won't make sense to the user, so I think we need to stick with just "saved" for now even if it's not quite right
```suggestion
        ..showSaveStatus('Saved!', autoHide: true);
```